ca/config.go

```diff
@@ -275,8 +340,8 @@ func NewConfigPaths(baseCertDir string) *SecurityConfigPaths {
 	}
 }
 
-// GenerateJoinToken creates a new join token.
-func GenerateJoinToken(rootCA *RootCA) string {
+// GenerateJoinToken creates a new join token.  If FIPS is enabled, the join token is of a new format.
```

Comment:
this is prob a dumb question, but can you please clarify what you mean by 'a new format' ?